src/xtopcom/xnode/xconfig.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "json/json.h"
+
+
+
+namespace top {
+    namespace topio {
+
+    class xconfig {
```

Comment:
what's the difference between the config object defined in `xconfig` and the xconfig in `xnode`?